iFixit/pim-community-standard
# Update docker compose usages

## Summary
Same as https://github.com/iFixit/valkyrie/pull/1128, this updates `docker-compose` usages to `docker compose`.

## QA notes
I am not sure what's the best way to test this. We can try the following:
1. Repeat the steps from [here](https://github.com/iFixit/server-templates/blob/76eb62d14f1d5a2a9a6313e5393c0d184b917674/specific/akeneo/startup-scripts/install-akeneo#L22-L31).
2. Then run `make dev` to confirm dev version is working?


closes https://github.com/iFixit/ifixit/issues/54794
